### DISCUSSION TITLE
New Seedstudio reComputer Jetson-10-1-A0 - Dual TPUs?

### DISCUSSION THREAD

**Discussion by sibero80** [blakeblackshear/frigate#3236] [General]
Seedstudio just reeleased this board: https://www.seeedstudio.com/Jetson-10-1-A0-p-5336.html
After writing to their support team, they confirmed the M.2 Key-e Slot supports 2x PCIe Lanes.

My planned use case for Frigate is around having presence detection in my home, so 1 TPU would go to object recognition and I´m thinking on using the 2nd TPU for Face Recognition.

I´m interested on what price/benefit balance you think this setup has?
Any benefits of the Nvidia Accelerator? I'll have 8x 720P cameras.

**Comment by kbrouder** [comment]
I have a similar carrier board from waveshare, I didn't spend a lot of time trying, but in that E-Key slot I only got a single TPU to be recognized...

I am in a similar boat, currently still using frigate on cpu detectors on my server, running 11 cameras 720P-4k, but have the detection streams on fairly low res. Its not ideal, but havent been able to find a solution where I can use my dual edge coral and still have enough CPU/GPU to decode well...
